arangod/CMakeLists.txt

```diff
@@ -967,6 +967,7 @@ target_link_libraries(arango_iresearch arango_indexes)
 target_link_libraries(arango_iresearch arango_cluster_engine)
 
 target_link_libraries(arango_metrics arango)
+target_link_libraries(arango_metrics arango::validation)
```

Comment:
Wondering what this change has to do with the metrics UI changes... :thinking:

Reply:
Without that addition, the target arango_metrics is not able to compile.